pages/api/cron/bookingReminder.ts

```diff
@@ -0,0 +1,74 @@
+import type { NextApiRequest, NextApiResponse } from "next";
```

Comment:
Not sure how this works. Vercel doesn't support cron jobs natively. 

Secondly req.query.apiKey equals process.env.CRON_API_KEY checks what exactly?

Reply:
This comment is about the entire file in general, not just the import.

Reply:
It is just a normal endpoint, that is called by an external scheduler

Reply:
Ahh okay, who's providing that?